src/lib/autocomplete/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+package(default_visibility=["//visibility:public"])
+load("@angular//:index.bzl", "ng_module")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_library")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_binary")
+
+
+ng_module(
+  name = "autocomplete",
+  srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
+  module_name = "@angular/material/autocomplete",
+  assets = [":autocomplete_css"],
+  deps = [
+    "//src/lib/core",
+    "//src/lib/form-field",
+    "//src/cdk/a11y",
+    "//src/cdk/bidi",
+    "//src/cdk/collections",
+    "//src/cdk/keycodes",
+    "//src/cdk/portal",
+    "//src/cdk/overlay",
+  ],
+  tsconfig = ":tsconfig-build.json",
+)
+
+
+sass_binary(
+  name = "autocomplete_scss",
+  src = "autocomplete.scss",
+  deps = ["//src/lib/core:core_scss_lib"],
+)
+
+# TODO(jelbourn): remove this when sass_binary supports specifying an output filename and dir.
+# Copy the output of the sass_binary such that the filename and path match what we expect.
+genrule(
+  name = "autocomplete_css",
+  srcs = [":autocomplete_scss"],
+  outs = ["autocomplete.css"],
+  cmd = "cat $(locations :autocomplete_scss) > $@",
```

Comment:
since you have a single src you can
cmd = "cp $< $@"

Reply:
also, you do this enough that you might want to make a macro around sass_binary...

Reply:
The sass_binary rule claims that it has multiple source even though there is just one